src/app/workflow/permissions/permissions.component.ts

```diff
@@ -76,6 +76,7 @@ export class PermissionsComponent implements OnInit {
         .addWorkflowPermission(this.workflow.full_workflow_path, { email: value, role: permission }, this.entryType === EntryType.Service)
         .subscribe(
           (userPermissions: Permission[]) => {
+            this.alertService.detailedSuccess('Permissions updated');
```

Comment:
Should we add similar calls to the alert service in the `remove()` method, also?

Reply:
Added